pkg/metastore/hive/ms_client.go

```diff
@@ -41,6 +46,36 @@ func NewMSClient(ctx context.Context, addr string, secure bool) (*MSClient, erro
 	return msClient, nil
 }
 
+func CopyOrMergeAll(ctx context.Context, fromClient, toClient MSClient, schemaFilter, tableFilter, toBranch string) error {
```

Comment:
`fromClient` and `toClient` structures: consider passing them by reference and not a copy.

Reply:
👍